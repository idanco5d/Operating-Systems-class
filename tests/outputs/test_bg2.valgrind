==7718== Memcheck, a memory error detector
==7718== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==7718== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==7718== Command: /mnt/c/Users/illus/os/hw/1/smash
==7718== Parent PID: 7709
==7718== 
==7718== 
==7718== FILE DESCRIPTORS: 4 open (3 std) at exit.
==7718== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_bg2.valgrind
==7718==    <inherited from parent>
==7718== 
==7718== 
==7718== HEAP SUMMARY:
==7718==     in use at exit: 0 bytes in 0 blocks
==7718==   total heap usage: 160 allocs, 160 frees, 86,313 bytes allocated
==7718== 
==7718== All heap blocks were freed -- no leaks are possible
==7718== 
==7718== For lists of detected and suppressed errors, rerun with: -s
==7718== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
