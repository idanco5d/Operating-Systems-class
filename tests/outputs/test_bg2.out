smash> submitted by Eran
smash> smash> smash: got ctrl-Z
smash: process 8869 was stopped
smash> smash> [1] sleep 100 : 8869 0 secs (stopped)
[2] sleep 200& : 8939 0 secs
smash> smash> smash> smash> sleep 100 : 0
smash> [1] sleep 100 : 8869 0 secs
[2] sleep 200& : 8939 0 secs
smash> smash: got ctrl-Z
smash: process 8940 was stopped
smash> smash: got ctrl-Z
smash: process 8983 was stopped
smash> smash> sleep 400 : 0
smash> [1] sleep 100 : 8869 1 secs
[2] sleep 200& : 8939 1 secs
[3] sleep 300 : 8940 0 secs (stopped)
[4] sleep 400 : 8983 0 secs
[5] sleep 500& : 9030 0 secs
smash> smash: sending SIGKILL signal to 5 jobs:
8869: sleep 100
8939: sleep 200&
8940: sleep 300
8983: sleep 400
9030: sleep 500&
